gitlab/README.md

```diff
@@ -10,6 +10,12 @@ See [Monitoring GitLab with Prometheus][1] for more information.
 
 ## Setup
 
+### Prerequisites
+
+This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see the [Latest and Legacy Versioning For OpenMetrics-based Integrations][17].
+
+Metrics marked as `[Legacy]`, `[OpenMetricsV1]`, or `[OpenMetricsV2]` are only available using the corresponding version of the GitLab integration. Metrics not marked are collected by all three versions.
```

Comment:
Is there any way to keep this note closer to where the metrics are listed (ideally, right before them).

Also, I think the term "Legacy" here is becoming overloaded and I think we'll be able to drop it (just opened #15296).

```suggestion
Metrics marked as `[OpenMetricsV1]` or `[OpenMetricsV2]` are only available using the corresponding version of the GitLab integration. All other metrics are collected by both versions.
```

Reply:
@alopezz We can't move the Metrics sentence in front of the magic sentence in the ## Metrics section otherwise it will not autogenerate the table of metrics unfortunately: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1169426657/Integrations#metadata.csv.